backend/questions/src/questions/questionRouter.ts

```diff
@@ -98,7 +134,12 @@ questionRouter.put(
     const updateData: QuestionUpdateData = {
       title: title || existingQuestion.title,
       body: body || existingQuestion.body,
-      complexity: complexity || existingQuestion.complexity
+      complexity: complexity || existingQuestion.complexity,
+      avgRating: ratings
+        ? (existingQuestion.ratings * existingQuestion.avgRating + ratings) /
+          (existingQuestion.ratings + 1)
+        : existingQuestion.avgRating,
+      ratings: ratings ? existingQuestion.ratings + 1 : existingQuestion.ratings
```

Comment:
We need to wrap the truthy check here to check if it is 0 too, or we can just close 2 eyes here and reject 0 ratings xD

Reply:
Hmmm shall we validate that rating is in [1, 5]?

Reply:
Yeah we could, then I'll take note of the ui similarly

Reply:
Hahaha I realise I still need to check if it's zero...Pushed the fix